tests/unit_tests/test_helm_chart.py

```diff
@@ -0,0 +1,228 @@
+import subprocess
```

Comment:
Shame there is no well maintained Helm python client

Reply:
Ultimately helm itself is a client, so the only way to do a Python version other than this is to reimplement it